ts/src/profilers/heap-profiler.ts

```diff
@@ -15,41 +15,52 @@
  */
 
 import {perftools} from '../../../proto/profile';
+import {defaultConfig} from '../config';
+
 import {serializeHeapProfile} from './profile-serializer';
 
 const profiler = require('bindings')('sampling_heap_profiler');
 
-export class HeapProfiler {
-  private enabled = false;
-
-  /**
-   * @param intervalBytes - average bytes between samples.
-   * @param stackDepth - upper limit on number of frames in stack for sample.
-   */
-  constructor(private intervalBytes: number, private stackDepth: number) {
-    this.enable();
-  }
+let enabled = false;
+let heapIntervalBytes = defaultConfig.heapIntervalBytes;
```

Comment:
I think you could initialize heapIntervalBytes and heapStackDepth to zeros here. Any reason to depend on config?

Reply:
done.